dataframe-jdbc/src/test/kotlin/org/jetbrains/kotlinx/dataframe/io/h2/h2Test.kt

```diff
@@ -967,4 +972,127 @@ class JdbcTest {
         }
         exception.message shouldBe "H2 database could not be specified with H2 dialect!"
     }
+
+    // helper object created for API testing purposes
+    object CustomDB: H2(MySql)
```

Comment:
If it's only for testing, i'd recommend to keep H2 final and use `object CustomDB : DbType by H2(MySql)`

Reply:
Delegation is supported only for interfaces.